autode/wrappers/base.py

```diff
@@ -1,13 +1,21 @@
+import os
+import numpy as np
 from abc import ABC
 from abc import abstractmethod
+from typing import Collection
 from shutil import which
 from autode.log import logger
 from autode.utils import requires_output
 from copy import deepcopy
-import os
 
 
-class ElectronicStructureMethod(ABC):
+class Method:
```

Comment:
whats this for?

Reply:
Was thinking about having a base class for molecular mechanics and electronic structure methods - nothing to do with this PR really

Reply:
ok